doc/usecases/bindings/UC_existing_application.md

```diff
@@ -0,0 +1,17 @@
+# Use Case: Programming Language Bindings in existing Application
+
+## Summary
+
+- **Scope:** API
+- **Level:** Underwater
+- **Actors:** API User
+- **Brief:** Developer wants to use libelektra in an existing application written in another language than C
+- **Status:** Draft
```

Comment:
Why is the status "draft"?
Aren't there already some existing and working bindings which can fulfill the use case?

```suggestion
- **Status:** Implemented
```